<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>750</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Tower Defense</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1500</width>
     <height>750</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1500</width>
     <height>750</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>750</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pageMenu">
     <property name="styleSheet">
      <string notr="true">QWidget#pageMenu {
    border-image: url(:/assets/img/homeBackground.jpg) 0 0 0 0 stretch stretch;
        background-repeat: no-repeat;
        background-position: center;
}</string>
     </property>
     <widget class="QPushButton" name="buttonExit">
      <property name="geometry">
       <rect>
        <x>645</x>
        <y>450</y>
        <width>240</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
        background-color: rgb(85, 0, 0);
    color: white;
    border: none;
    border-radius: 5px;
        font: 25pt &quot;Eras Medium ITC&quot;;
}

QPushButton:hover {
    background-color: #2980b9;  /* Couleur de fond en survol */
}
</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonGame">
      <property name="geometry">
       <rect>
        <x>645</x>
        <y>270</y>
        <width>240</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
        background-color: rgb(153, 0, 0);
        font: 25pt &quot;Eras Medium ITC&quot;;
    color: white;
    border: none;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #2980b9;  /* Couleur de fond en survol */
}
</string>
      </property>
      <property name="text">
       <string>Game</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>-20</y>
        <width>721</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt; font-weight:700;&quot;&gt;Tower Defense&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRules">
      <property name="geometry">
       <rect>
        <x>645</x>
        <y>360</y>
        <width>240</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
        background-color: rgb(130, 0, 0);
    color: white;
    border: none;
    border-radius: 5px;
        font: 25pt &quot;Eras Medium ITC&quot;;
}

QPushButton:hover {
    background-color: #2980b9;  /* Couleur de fond en survol */
}
buttonGame{
background-color: rgb(39, 174, 96);
}
</string>
      </property>
      <property name="text">
       <string>Rules</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageGame">
     <property name="styleSheet">
      <string notr="true">QWidget#pageGame {
    border-image: url(:/assets/img/gameBackground3.png) 0 0 0 0 stretch stretch;
    background-repeat: no-repeat;
    background-position: center;
}</string>
     </property>
     <widget class="QPushButton" name="buttonMenu">
      <property name="geometry">
       <rect>
        <x>1320</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="towerUpgrade1_1">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>290</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="towerUpgrade1_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>290</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="towerUpgrade1_3">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>410</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="towerButton1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>350</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="tower1">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>330</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="upgradeShop1">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>270</y>
        <width>221</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources.qrc">:/assets/img/circle.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelGold">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>690</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;500&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="goldRect">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>680</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources.qrc">:/assets/img/Sign.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="music">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources.qrc">:/assets/img/music.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="noMusic">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources.qrc">:/assets/img/noMusic.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonMonster1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="towerButton2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>560</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="towerUpgrade2_3">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>620</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="towerUpgrade2_1">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>500</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="towerUpgrade2_2">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>500</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="tower2">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>540</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="upgradeShop2">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>480</y>
        <width>221</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources.qrc">:/assets/img/circle.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lifeRect">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>680</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources.qrc">:/assets/img/Life.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelLife">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>690</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;500&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="imgButtonMonster1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources.qrc">:/assets/img/timer.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="timeLabel1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
    font-weight: 800;
    color: black;
        </string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPlay">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>22</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMute">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>24</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="baseAttacked">
      <property name="geometry">
       <rect>
        <x>715</x>
        <y>414</y>
        <width>200</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources.qrc">:/assets/img/base2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="waveCounterLabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
    font-weight: 800;
    color: black;
        </string>
      </property>
      <property name="text">
       <string>Wave 1 / 3</string>
      </property>
     </widget>
     <zorder>tower1</zorder>
     <zorder>upgradeShop1</zorder>
     <zorder>upgradeShop2</zorder>
     <zorder>buttonMenu</zorder>
     <zorder>goldRect</zorder>
     <zorder>labelGold</zorder>
     <zorder>music</zorder>
     <zorder>noMusic</zorder>
     <zorder>tower2</zorder>
     <zorder>towerButton2</zorder>
     <zorder>towerButton1</zorder>
     <zorder>towerUpgrade1_3</zorder>
     <zorder>towerUpgrade1_1</zorder>
     <zorder>towerUpgrade1_2</zorder>
     <zorder>towerUpgrade2_3</zorder>
     <zorder>towerUpgrade2_1</zorder>
     <zorder>towerUpgrade2_2</zorder>
     <zorder>lifeRect</zorder>
     <zorder>labelLife</zorder>
     <zorder>imgButtonMonster1</zorder>
     <zorder>ButtonMonster1</zorder>
     <zorder>timeLabel1</zorder>
     <zorder>btnPlay</zorder>
     <zorder>btnMute</zorder>
     <zorder>baseAttacked</zorder>
     <zorder>waveCounterLabel</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
